src/nodes.coffee

```diff
@@ -446,6 +446,20 @@ exports.NullLiteral = class NullLiteral extends Literal
 
 exports.BooleanLiteral = class BooleanLiteral extends Literal
 
+exports.InfinityLiteral = class InfinityLiteral extends Literal
+  constructor: ->
+    super 'Infinity'
+
+  compileNode: ->
+    [@makeCode '2e308']
+
+exports.NaNLiteral = class NaNLiteral extends Literal
+  constructor: ->
+    super 'NaN'
+
+  compileNode: ->
+    [@makeCode '0/0']
```

Comment:
Since this compilation includes an infix operator, we should either surround it with parens or make sure `NaNLiteral`s have the same precedence as `/`.